### DISCUSSION TITLE
GSoC 2024: Seeking Discussion on GSoC 2024 OpenVINO Projects

### DISCUSSION THREAD

**Discussion by inbasperu** [openvinotoolkit/openvino#23448] [Google Summer of Code]
Hello everyone,

I'm an undergraduate senior majoring in Electronics and Communication Engineering at NITK, expressing my keen interest in participating in Google Summer of Code 2024 with the OpenVINO Toolkit. My journey with OpenVINO began when I explored its capabilities for fast edge inference on FPGAs, sparking a deep interest in the toolkit. As an enthusiast in signal processing and machine learning, I find joy in the realms of mathematics and programming. Currently, my undergraduate thesis focuses on solving inverse problems in imaging, presenting a unique opportunity to merge my interests through contributions to OpenVINO, aiming to make a positive impact on the community.

### Projects of Interest

I am interested in contributing to the following projects:

- **Project 7**: [OpenVINO Adapters - PyTorch Adapter](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#7-openvino-adapters---pytorch-adapter)
- **Project 14**: [Add Time Series Anomaly Detection Support to Anomalib](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#14-add-time-series-anomaly-detection-support-to-anomalib)
- **Project 20**: [Eyes Health Utilizing OpenVINO and Neural Processing Unit (NPU) on an AI PC](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#20-eyes-health-utilizing-openvino-and-neural-processing-unit-npu-on-an-ai-pc)

### Contributions

I have already contributed to the OpenVINO community by getting a pull request merged ([PR #23063](https://github.com/openvinotoolkit/openvino/pull/23063#issue-2151650002)) and am currently working on another ([PR #22993](https://github.com/openvinotoolkit/openvino/pull/22993#issue-2146681772)).

I look forward to the opportunity of collaborating with members of this community and contributing to the continuous evolution of the OpenVINO Toolkit.

Best regards,
Inbasekaran

**Comment by mlukasze** [comment]
@jiwaszki :)

**Reply by adrianboguszewski** [reply]
Also @samet-akcay @djdameln @ashwinvaidya17 for project 14 and @DimaPastushenkov and @zhuo-yoyowz for project 20

**Reply by inbasperu** [reply]
I have come up with an initial proposal for Project 20 after doing some background research. Would it be possible to discuss the exact detailed requirements over email with @DimaPastushenkov and @zhuo-yoyowz?

**Reply by zhuo-yoyowz** [reply]
Sure. Happy to discuss over email. Feel free to send email to Dima and me. Thanks!

**Reply by inbasperu** [reply]
Dear Mentors @DimaPastushenkov and @zhuo-yoyowz,

I have sent you an email containing my proposal and am eagerly seeking your feedback. Your guidance would be invaluable to me. Could you please review my proposal? Thank you very much for your time and assistance.

**Reply by DimaPastushenkov** [reply]
HI @inbasperu , I haven't receive an email from you unfortunately. Did you use email address mentioned in my github account? Could you please double check. Thank, Dmitriy

**Reply by inbasperu** [reply]
Hello @DimaPastushenkov,

I've sent an email to dmitriy.pastushenkov@intel.com and zhuo.wu@intel.com, and I've also shared the Google Doc with both addresses. Could you please check your spam folder? It's possible that my email was directed there.

Thank you.

**Reply by inbasperu** [reply]
Additionally, I've just sent another follow-up email to the same addresses.
